specification/storage/resource-manager/Microsoft.Storage/stable/2023-05-01/storage.json

```diff
@@ -0,0 +1,5811 @@
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/Resource"
+        }
+      ],
+      "description": "The local user associated with the storage accounts."
+    },
+    "LocalUserProperties": {
+      "type": "object",
+      "properties": {
+        "permissionScopes": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PermissionScope"
+          },
+          "description": "The permission scopes of the local user."
+        },
+        "homeDirectory": {
+          "type": "string",
+          "description": "Optional, local user home directory."
+        },
+        "sshAuthorizedKeys": {
+          "$ref": "#/definitions/SshAuthorizedKeys"
+        },
+        "sid": {
+          "readOnly": true,
+          "type": "string",
+          "description": "A unique Security Identifier that is generated by the server."
+        },
+        "hasSharedKey": {
+          "type": "boolean",
+          "description": "Indicates whether shared key exists. Set it to false to remove existing shared key."
+        },
+        "hasSshKey": {
+          "type": "boolean",
+          "description": "Indicates whether ssh key exists. Set it to false to remove existing SSH key."
+        },
+        "hasSshPassword": {
+          "type": "boolean",
+          "description": "Indicates whether ssh password exists. Set it to false to remove existing SSH password."
+        },
+        "userId": {
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32",
+          "description": "A unique Identifier that is generated by the server."
+        },
+        "groupId": {
+          "type": "integer",
+          "format": "int32",
+          "description": "An identifier for associating a group of users."
+        },
+        "allowAclAuthorization": {
+          "type": "boolean",
+          "description": "Indicates whether ACL authorization is allowed for this user. Set it to false to disallow using ACL authorization."
+        }
+      },
+      "description": "The Storage Account Local User properties."
+    },
+    "LocalUserKeys": {
+      "type": "object",
+      "properties": {
+        "sshAuthorizedKeys": {
+          "$ref": "#/definitions/SshAuthorizedKeys"
+        },
+        "sharedKey": {
+          "x-ms-secret": true,
+          "$ref": "#/definitions/SharedKey"
+        }
+      },
+      "description": "The Storage Account Local User keys."
+    },
+    "LocalUserListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LocalUser"
+          },
+          "description": "List of local users requested.",
+          "readOnly": true
```

Comment:
we   should be able to remove this readOnly entry here. the previous list result didnt have it and it is flagged as a breaking change. i think i copied it from a similar list result example and it isnt required